scrapy/http/request/json_request.py

```diff
@@ -36,6 +40,12 @@ def __init__(self, *args, **kwargs):
         self.headers.setdefault('Content-Type', 'application/json')
         self.headers.setdefault('Accept', 'application/json, text/javascript, */*; q=0.01')
 
+    @property
+    def dumps_kwargs(self):
+        if self._dumps_kwargs is None:
+            self._dumps_kwargs = {}
```

Comment:
Looking at [the coverage results](https://app.codecov.io/gh/scrapy/scrapy/compare/5130/diff#D5L45) and then at `__init__` above, it seems these 2 lines are unnecessary.